edudoc/competitor-tutorial/tutorial.md

```diff
@@ -3,7 +3,8 @@
 Scroll to the bottom of the document for translations.
 
 ## WCA Organization - General Introduction
-![WCA logo](../../assets/WCAlogo_notext.svg){.logo height=133px}
+![]({logo}){.logo height=133px}
```

Comment:
Tooltip `WCA logo` is gone...?

Reply:
@gregorbg wait, that's a tooltip? On my render it just showed up as text. Why is it different for me?

Reply:
@gregorbg I don't think it is, I just checked on the documents that's on the website right now

Reply:
A tooltip is the small box that pops up next to your mouse cursor if you hover over an element. Probably depends on the PDF viewer whether you can see it.

In any case, I'd prefer if you leave the information that is currently in the documents and don't modify it, even if it might seem redundant in your viewer.
We may choose to remove redundant tooltips in another PR, it is simply not part of the purpose of this commit.

Reply:
Sure, I'll put it back, I just assumed that because my builder is rendering it as text, it would work the same in production